VoteCounter/src/VoteCount.cs

```diff
@@ -74,7 +108,25 @@ private void findVotes(Post post)
 
         private bool isVoteValid(string vote)
         {
-            return playerList.Contains(vote);
+            foreach(Player p in playerList)
+            {
+                //check to see if the player being voted for is on the playerlist
+                if (String.Compare(vote, p.mainName,true) == 0)
+                    return true;
+                if (p.nicknameList.Count > 0)
+                {
+                    for (int i = 0; i < p.nicknameList.Count; i++)
+                    {
+                        //or if it has a nickname in the list
+                        bool r = String.Compare(p.nicknameList[i], vote, true) == 0;
+                        string Q = p.nicknameList[i];
+                        if (String.Compare(vote, p.nicknameList[i], true) == 0)
+                            return true;
+                    }
+                }
+            }
+            return false;
+            //return playerList.Contains(vote);
```

Comment:
Don't worry about that. If it's code you don't need, just remove it. It's still very visible in the diffs so it won't be lost or anything. Rest assured I'll tell you if you remove something you shouldn't have :P